modules/olm-creating-fb-catalog-image.adoc

```diff
@@ -3,17 +3,17 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 
 ifdef::openshift-origin[]
-:registry-image: quay.io/openshift/origin-operator-registry:4.9.0
+:registry-image: quay.io/operator-framework/opm:latest
 endif::[]
 ifndef::openshift-origin[]
-:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v4.9
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v{product-version}
```

Comment:
This is because it's based on a working/feature branch (`opm_413` here), but when it lands in a proper versioned branch (e.g., `enterprise-4.13`), it will be replaced with the correct number (e.g., `4.13`).